### PR TITLE
Documentation website landing page

### PR DESCRIPTION
**Pull request by bharati-21**
Styled and created a simple landing page. It includes -
- Navbar section
- Header with hero text and hero image, CTA buttons for getting started and documentation websites
- An introduction section for the library

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,34 +13,135 @@
     <body>
         <div class="grid-container">
             <nav class="navbar flex--row flex--justify-between flex--align-center">
-                <a href="/" class="brand flex--row flex--align-center">
+                <a href="/" class="brand flex--row flex--align-center flex--justify-center">
                     <img 
-                        src="./logo-assets/elixir-logo.png" 
+                        src="../logo-assets/elixir-logo.png"
```
Comment (sonishreyas) [inline index.html:18]: Just a suggestion. You can use the GitHub raw link for images so even if you want to use it somewhere else you wouldn't have to keep the logo-asset folder in that project

**Review by sonishreyas — APPROVED** [review]
Looks good to me

File: styles.css [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 
 /* grid-container */
+:root {
+    --navbar-height: 10vh;
+}
 
 .grid-container {
     display: grid;
-    grid-template-columns: 250px 1fr 1fr 1fr 1fr 1fr;
+    grid-template-columns: auto 1fr 1fr 1fr 1fr 1fr;
```
Comment (yashpatel1797) [inline styles.css:10]: here you can use repeat(5, 1fr).

**Review by yashpatel1797 — APPROVED** [review]
Looks good to me.

File: index.html (hunk 1 above)
Comment (bharati-21) [reply index.html:18]: Okay, I shall do this if i have the bandwidth before submission
Thank you for your time and review 👍🏽

File: styles.css (hunk 2 above)
Comment (bharati-21) [reply styles.css:10]: Yes, done. 
Thank you for your time and review! 👍🏽

**Pull request merged**